.travis.yml

```diff
@@ -44,4 +44,6 @@ install:
   - docker-compose pull
 
 script:
-  - docker-compose run --use-aliases -e BOULDER_CONFIG_DIR="${BOULDER_CONFIG_DIR}" -e RUN="${RUN}" -e TRAVIS="${TRAVIS}" -e TRAVIS_PULL_REQUEST="${TRAVIS_PULL_REQUEST}" -e COVERALLS_TOKEN="${COVERALLS_TOKEN}" ${CONTAINER:-boulder} ./test.sh
+  # Export all variables with a TRAVIS prefix
+  - exported_vars=(BOULDER_CONFIG_DIR RUN "${!TRAVIS@}" COVERALLS_TOKEN)
+  - docker-compose run --use-aliases "${exported_vars[@]/#/-e}" ${CONTAINER:-boulder} ./test.sh
```

Comment:
This is some cool Bash magic, but I don't understand it well enough to maintain it, and I think other readers are likely to be confused by it. Let's keep the boring, repetitive version we had before, and just add back the vars that were incorrectly removed. We could clean it up a bit by putting each -e on a line by itself with `\`.

Reply:
Fair enough. I grepped goveralls to find out all travis variables it looks at and added those. Instead of \ I just used a yaml folded block scalar.